src/main/java/slimeknights/mantle/recipe/data/BaseRecipeProvider.java

```diff
@@ -0,0 +1,338 @@
+package slimeknights.mantle.recipe.data;
+
+import net.minecraft.advancements.ICriterionInstance;
+import net.minecraft.data.*;
```

Comment:
No wildcard imports please, makes code harder to trace on Github.

Reply:
Dang, missed that one. I need to see about getting idea to not do that.